README.md

```diff
@@ -47,80 +47,12 @@ ORM techniques, this does not rely on annotation or reflection.
 
 Until we write thorough documentation, the best place to understand
 what you can do is to go through the [test
-suite](https://github.com/w3c/banana-rdf/tree/series/0.8.x/rdf-test-suite).
+suite](https://github.com/w3c/banana-rdf/tree/series/0.8.x/rdf-test-suite) and read [banana-rdf wiki](https://github.com/banana-rdf/banana-rdf/wiki).
 
-How to start geeking
---------------------
+Documentation
```

Comment:
In general, I think this is good. I wonder if we need a very short getting started section before this, for example :

#Getting Started

Banana-rdf is currently available for Scala 2.10 and 2.11, and for scala-js on both versions.

To get started with SBT, simply add the following to your build.sbt file: 
For scala on the JVM:
```
libraryDependencies += "org.w3" %% "banana-rdf" % "0.8.1"
``` 

For JS:
```
libraryDependencies += "org.w3" %%% "banana-rdf" % "0.8.1"
```